copter/source/docs/tuning-process-instructions.rst

```diff
@@ -70,6 +190,8 @@ The first take off is the most dangerous time for any multirotor. Care must be t
 10. Hover at approximately 1m altitude and apply small (5 degrees) control inputs into roll and pitch.
 11. Immediately land if any oscillation is observed.
 
+# TODO what to do in case of oscillation ?
+
```

Comment:
I don't know that there is much more we can say other than "Immediately land".

Tuning instructions in this situation are further down.

Reply:
ok, so I just add Next section will explain how to remove the oscillations.